tests/test-update-install-flatpaks.c

```diff
@@ -2049,6 +2049,127 @@ test_update_deploy_flatpaks_on_reboot (EosUpdaterFixture *fixture,
   g_assert (g_strv_contains ((const gchar * const *) deployed_flatpaks, flatpaks_to_install[0].app_id));
 }
 
+/* Insert a list of flatpaks to automatically uninstall on the commit
+ * and simulate a reboot by running eos-updater-flatpak-installer. This
+ * should check the deployment for a list of flatpak actions and
+ * uninstall them in the system installation */
+static void
+test_update_uninstall_flatpaks_on_reboot (EosUpdaterFixture *fixture,
+                                       gconstpointer      user_data)
+{
+  g_auto(EtcData) real_data = { NULL, };
+  EtcData *data = &real_data;
+  FlatpakToInstall flatpaks_to_install[][2] = {
+    {
+      { "install", "com.endlessm.TestInstallFlatpaksCollection", "test-repo", "org.test.Test", "app", FLATPAK_TO_INSTALL_FLAGS_NONE }
+    },
+    {
+      { "install", "com.endlessm.TestInstallFlatpaksCollection", "test-repo", "org.test.Test", "app", FLATPAK_TO_INSTALL_FLAGS_NONE },
```

Comment:
Nitpick: Could do with a comment about this list being append-only from the last one.

(I’ll fix this before pushing.)

Reply:
(Fixed.)